src/pudl/analysis/timeseries_cleaning.py

```diff
@@ -882,7 +883,7 @@ def median_of_rolling_median_offset(
         warnings.filterwarnings(
             "ignore", category=RuntimeWarning, message="All-NaN slice encountered"
         )
-        return np.nanmedian(shifted, axis=0)
+        return bn.nanmedian(shifted, axis=0)
```

Comment:
It will probably be hard to beat this for "memory reduction per characters changed"